jenkinsfiles/Jenkinsfile.grobid

```diff
@@ -0,0 +1,25 @@
+elifePipeline {
+    node('containers-jenkins-plugin') {
+        def commit
+
+        stage "Checkout", {
+            sh "rm -rf grobid && git clone -b catchall git@github.com:elifesciences/grobid.git"
+            commit = sh(returnStdout: true, script: "cd grobid && git rev-parse HEAD").trim()
+            echo "commit: ${commit}"
+        }
+
+        stage "Build Image", {
+            dir('grobid') {
+                // we are using a blank version as the commit hash wouldn't be a valid version
+                sh "docker build -t elifesciences/grobid:${commit} --build-arg GROBID_VERSION= ."
+            }
+        }
+
+        stage 'Push unstable image', {
+            def image = DockerImage.elifesciences(this, 'grobid', commit)
+            def unstable_image = image.addSuffixAndTag('_unstable', commit)
+            unstable_image.tag('latest').push()
```

Comment:
should these two tags be pushed in the inverse order? First the commit, then `latest`, then any separate tag corresponding to a Git tag.

Reply:
Not sure if I should take the hint. :-) You mentioned it on most PRs.
It is though what was agreed on initially.
Happy to change the order as it would make sense, but we should probably apply that to all of the projects consistently?

Reply:
(Created an [issue](https://github.com/elifesciences/issues/issues/4996) to track it instead)

Reply:
This continues to change depending on the phase of the Moon...